### PR TITLE
Prepares return of unit tests

### PR DESCRIPTION
**Pull request by Itee**
The three first commit are the "key".

Re-allow units testing on all three files and modules, and prepare them for the future !

Currently it run all editor and src tests that already exist (some bugs in src appear !!!)

Enjoy,
Tristan

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   },
   "scripts": {
     "build": "rollup -c",
+    "build-test": "rollup -c rollup.unit.config.js",
```
Comment (mrdoob) [inline package.json]: Hmm, I think there is a way to only use one rollup.config.js and pass parameters.

Comment (TristanVALCKE) [inline package.json]: Ok, i will try to pass parameters instead using an other config file.

**Comment by TristanVALCKE** [comment]
@mrdoob Done with a single file

**Comment by mrdoob** [comment]
Actually... Do you mind reverting to two files and move `rollup.unit.config.js` to `test/rollup.config.js`?

**Comment by TristanVALCKE** [comment]
I could reverting to two files and move rollup.unit.config.js to test/rollup.config.js, or keep a single file with builds parameter but...

What confusing you ?

A single file is easy to maintain, and i think the builds parameter code is easy to understand and is easy to extend.

Two file will keep different things in different places...

But what do you prefer, or plan to do with config files ???

**Comment by mrdoob** [comment]
> A single file is easy to maintain, and i think the builds parameter code is easy to understand and is easy to extend.

I find the single file a bit too complicated, considering I rarely run the tests.

And, again, because I rarely run them, I don't think the tests deserve an additional file in the root.

**Comment by TristanVALCKE** [comment]
Reverted, and move rollup config file under test. I keep "unit" in the file name to avoid any confusion with the main config file, are you agree with that ?

**Comment by mrdoob** [comment]
Looks good!

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Wait, why isn't this using the builds in `/build/`?

**Pull request merged**